static/app/views/organizationStats/getReasonGroupName.ts

```diff
@@ -91,7 +118,37 @@ const invalidReasonsGroup: Record<string, DiscardReason[]> = {
     DiscardReason.INVALID_REPLAY_VIDEO,
   ],
   payload: [DiscardReason.PAYLOAD, DiscardReason.INVALID_COMPRESSION],
-  too_large: [DiscardReason.TOO_LARGE],
+  too_large: [
+    DiscardReason.TOO_LARGE, // Left for backwards compatibility
+    DiscardReason.TOO_LARGE_UNKNOWN,
+  ],
```

Comment:
```suggestion
```

since we are going to expose most of these reasons, we could remove the `too_large` here and in the function `getInvalidReasonGroupName` do the following:

```
function getInvalidReasonGroupName(reason: DiscardReason): string {
  if (reason.startsWith(DiscardReason.TOO_LARGE)) {
    if ([DiscardReason.TOO_LARGE, DiscardReason.TOO_LARGE_UNKNOWN].includes(reason)) {
      return startCase(DiscardReason.TOO_LARGE);
    }
    return startCase(reason);
  }

  for (const [group, reasons] of Object.entries(invalidReasonsGroup)) {
    if (reasons.includes(reason)) {
      return group;
    }
  }
  return 'internal';
}
```